webpack/assets/javascripts/bundle.js

```diff
@@ -30,6 +29,7 @@ import compute from './foreman_compute_resource';
 import componentRegistry from './react_app/components/componentRegistry';
 import i18n from './react_app/common/I18n';
 import * as foremanDocument from './react_app/common/document';
+import hosts from './foreman_hosts';
```

Comment:
Why not to import all of the dependencies, but just one? Looks bit weird now.

Reply:
this allow to consume the functions both via tfm.hosts.. and as a simple import {x} from ...

I did it in order to not break code base, and its a pattern we already have (e.g. see compute resources).

Reply:
oh, ok, make sense, thanks for explanation :+1: